docs/user-manual.rst

```diff
@@ -915,6 +915,108 @@ will compute a boolean that tells you whether the field "000000" passes
 a Chi-square test for two degrees of freedom with significance level
 0.05.
 
+
+Fuzzy logic
+-----------
+
+Flatline provides some functions to work with Fuzzy logic datasets.
```

Comment:
i'd rather say fuzzy logic features, fields or values, the full dataset need not
be about fuzzy logic